### DISCUSSION TITLE
yolov8 vs yolov5 accuracy when using tracking

### DISCUSSION THREAD

**Discussion by Christoffel21** [ultralytics/ultralytics#7828] [General]
Hi guys

I've tested the examples of yolov5 and yolov8 on the same video and the yolov5 detect more persons than the yolov8.
I used the code from the following repositories

Yolov5
https://github.com/RizwanMunawar/yolov5-object-tracking

Yolov8
https://github.com/ultralytics/ultralytics/tree/main/examples/YOLOv8-Region-Counter

1st picture yolov5n.pt 
2nd picture yolov8n.pt
3rd  picture yolov8s.pt
4th picture yolov8m.pt
![yolov5n](https://github.com/ultralytics/ultralytics/assets/50450102/8a5fea89-f6d3-4f0f-8ce4-2ec41665fe75)
yolov8m.pt

![yolov8n](https://github.com/ultralytics/ultralytics/assets/50450102/fbaab494-f1ae-499a-ab1f-1a049a2ae3bb)
![yolov8s](https://github.com/ultralytics/ultralytics/assets/50450102/960cc925-4434-42a1-8a66-3c58ae41a0b3)
![yolov8m](https://github.com/ultralytics/ultralytics/assets/50450102/eb48cd8c-eb8a-4855-8662-5868a75bb957)

**Comment by Christoffel21** [comment]
Would have thought yolov8 would out perform yolov5???

**Reply by pderrenger** [reply]
@Christoffel21 hello! It's great to hear that you're experimenting with both YOLOv5 and YOLOv8 for object tracking. The performance of these models can vary depending on a number of factors, including the specific configurations and weights used, the dataset they were trained on, and the version of the models.

If YOLOv8 is detecting fewer objects than YOLOv5 in your tests, it might be worth exploring the confidence thresholds, non-maximum suppression (NMS) settings, or even the possibility of further tuning the YOLOv8 model on your specific dataset. Each YOLO version has its own architectural differences and optimizations that might affect performance on certain tasks or datasets.

For tracking, YOLOv8 supports various modes and trackers, such as BoT-SORT and ByteTrack, which can be configured to suit your needs. It's also possible that YOLOv8's default settings are more conservative in terms of detections, prioritizing precision over recall.

We recommend reviewing the tracking configurations, experimenting with different thresholds, and potentially retraining the model with hyperparameter tuning to optimize for your specific use case. If you have further questions or need assistance, feel free to reach out to our community or open an issue on the Ultralytics YOLOv8 repo. We're here to help! 😊🚀
